frontend/tailwind.config.js

```diff
@@ -23,7 +24,34 @@ export default {
                 200: '#1C2227',
             },
         },
-        extend: {},
+        fontSize: {
+            xs: '0.75rem',
+            sm: '0.875rem',
+            base: '1rem',
+            xl: '1.25rem',
+            '2xl': '1.375rem',
+            '3xl': '1.75rem',
+            '4xl': '2.625rem',
+        },
+        fontWeight: {
+            normal: '400',
+            semibold: '500',
+            bold: '600',
+        },
+        lineHeight: {
+            '3': '1rem',
+            '4': '1.25rem',
+            '5': '1.375rem',
+            '6': '1.75rem',
+            '7': '2.125rem',
+            '8': '2.25rem',
+            '9': '3.5856rem',
+        },
+        extend: {
+            fontFamily: {
+                'sans': ['"Manrope"'],
```

Comment:
I believe it's necessary to have a fallback option. Previously, there were default fonts from TW, I think it was a nice option for fallback.